VIRTUAL_VACATION/vacation.pl

```diff
@@ -536,47 +556,53 @@ sub send_vacation_email {
         my $body = $row[1];
         my $from = $email;
         my $to = $orig_from;
-        my %smtp_connection;
-        %smtp_connection = (
-            'smtp' => $smtp_server,
-            'port' => $smtp_server_port,
-            'auth' => $smtp_auth,
-            'authid' => $smtp_authid,
-            'authpwd' => $smtp_authpwd,
-            'tls_allowed' => $smtp_tls_allowed,
-            'smtp_client' => $smtp_client,
-            'skip_bad_recipients' => 'true',
-            'encoding' => 'Base64',
-            'ctype' => 'text/plain; charset=UTF-8',
-            'headers' => 'Precedence: junk',
-            'headers' => 'X-Loop: Postfix Admin Virtual Vacation',
-            'on_errors' => 'die', # raise exception on error
-        );
-        my %mail;
-        %mail = (
-            'subject' => encode_mimewords($subject, 'Charset', 'UTF-8'),
-            'from' => $from,
-            'fake_from' => $friendly_from . " <$from>",
-            'to' => $to,
-            'msg' => encode_base64(encode("UTF-8", $body))
+
+		my $smtp_params = {
+			host => $smtp_server,
+            port => $smtp_server_port,
+            ssl_options => $ssl_options,
+			ssl  => $smtp_ssl,
+			timeout => $smtp_timeout,
+			localaddr => $smtp_client,
+            debug => 0,
+		};
+
+		if($smtp_authid ne ''){
+			$smtp_params->{sasl_username}=$smtp_authid;
+			$smtp_params->{sasl_password}=$smtp_authpwd;
+            $logger->info("Doing SASL Authentication with user $smtp_params->{sasl_username}\n");
+		};
+		
+        my $transport = Email::Sender::Transport::SMTP->new($smtp_params);
+		
+		$email = Email::Simple->create(
+            header => [
+                To      => $to,
+                From    => $from,
+                Subject => $subject,
```

Comment:
Because i didn't read the RFCs and tests with non ASCII-chars in the Subject worked with my client.   I'll revert that change.